src/environment.js

```diff
@@ -147,8 +152,7 @@ var Environment = Obj.extend({
                 }
 
                 // Resolve name relative to parentName
-                if (parentName && (name.indexOf("./") == 0 ||
-                                   name.indexOf("../") == 0)) {
```

Comment:
Yes, I'm looking at this, so adding a `isRelative` method to the loader is a best idea (to avoid this: https://github.com/mozilla/nunjucks/issues/375), will that be good?